lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -477,8 +478,52 @@ def __get_template_with_version(self):
 
         return template
 
+    def __get_storage_domain(self, disk_id):
+        storage_domain = None
+
+        disks = self._module.params.get('disks', list())
+
+        for disk in disks:
+            if disk_id == disk.get('id'):
```

Comment:
the plan now is: 
1. if there is a template specified and a storage domain: all disks related to that template will go into the storage domain
2. if there is a template specifeid and disks is specified each disk will go to the specified storage domain:
template: my_template
disks:
  - id_of_disk
  - storage domain 


@machacekondra please ack